src/quilla/__init__.py

```diff
@@ -63,8 +63,18 @@ def make_parser() -> argparse.ArgumentParser:  # pragma: no cover
         action='store',
         metavar='run_id',
         default=None,
-        help='A run ID for quilla, if manually passed in.'
-        'Used to set many quilla tests to have the same run ID'
+        help='A run ID for quilla, if manually passed in. '
+        'Used to set many quilla tests to have the same run ID '
+        'If no run ID is manually passed in, it will be auto-generated'
+    )
+    config_group.add_argument(
+        '-u',
+        '--update-baseline',
+        dest='update_baseline',
+        action='store_true',
+        help='Used to update the baseline images for VisualParity, '
+        'overwriting existing ones and creating new ones if necessary. '
```

Comment:
Create an issue/enhanchment to implement baseline versioning so we never overwrite baseline images. We should still update baseline image when it's missing when update_baseline is set.